pkg/controller/telemetry/utils.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 var (
-	log            = logger.NewLoggerField("pkg/telemetry")
+	log            = logger.NewLoggerScope("pkg/telemetry")
```

Comment:
```suggestion
	log            = logger.NewLoggerScope("telemetry")
```

or controller/telemetry